### PR TITLE
Root action to move received contributions between tiers

### PR DESCRIPTION
**Pull request by Betree**
Resolve https://github.com/opencollective/opencollective/issues/5000
Require https://github.com/opencollective/opencollective-api/pull/7053

### CODE REVIEW

File: components/root-actions/MoveReceivedContributions.js [hunk 1]
```diff
@@ -0,0 +1,267 @@
...
+      tier {
+        id
+        legacyId
+      }
+      fromAccount {
+        id
+        name
+        slug
+        isIncognito
+        imageUrl(height: 48)
```
Comment (SudharakaP) [inline components/root-actions/MoveReceivedContributions.js:44]: I think these two aren't used/changed and can be removed? 🤔

File: components/root-actions/MoveReceivedContributions.js [hunk 2]
```diff
@@ -0,0 +1,267 @@
...
+            name
+          }
+        }
+      }
+    }
+  }
+`;
+
+const getCallToAction = (selectedOrdersOptions, newTier) => {
+  const base = `Move ${selectedOrdersOptions.length} contributions`;
```
Comment (SudharakaP) [inline components/root-actions/MoveReceivedContributions.js:75]:
```suggestion
  const base = `Move ${selectedOrdersOptions.length} contribution(s)`;
```

**Review by SudharakaP — COMMENTED** [review]
@Betree : In addition to my comments above, one other thing I noticed is that the `ADDED_FUNDS` also show up in the select contributions picker? I don't think we should allow to move them right, only contributions? 🤔

File: pages/root-actions.js [hunk 3]
```diff
@@ -28,6 +29,12 @@ const MENU = [
     It can be used to mark contributions as "Incognito" by picking the contributor's incognito profile in "Move to".
     The payment methods used for the contributions are re-affected to the new profile, so make sure they have the permission to use them.`,
   },
+  {
+    id: 'Move received contributions',
+    Component: MoveReceivedContributions,
+    description: `This tool is meant to edit the account that received a contribution.
+    Use it to move contributions to different tiers, sub-projects, events, etc.`,
```
[the remark this answers is not in the document]
Comment (Betree) [reply pages/root-actions.js:36]: Not yet, it has been simplified for MVP. But that's the goal

**Comment by Betree** [comment]
> `ADDED_FUNDS`

Well, it does not work yet because the API does not support moving orders when the payment method is an account balance, but technically there's no limitation on the frontend side for this. Especially now that added funds support specifying a tier (https://github.com/opencollective/opencollective/issues/4009).

Note that, due to this feature being used only by admins, our requirements in terms of UX/i18n are very low (while there's no compromise on API tests, since these are powerful tools).

**Pull request merged**